### PR TITLE
boards: shield: add arceli w5500 ethernet

### PR DESCRIPTION
**Pull request by parthitce**
add shield for arceli w5500 ethernet breakout with SPI
serial bus

Signed-off-by: Parthiban Nallathambi <parthiban@linumiz.com>

### CODE REVIEW

**Review by rlubos — APPROVED** [review]
Thanks

File: boards/shields/arceli_eth_w5500/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Programming
+***********
+
+Set ``-DSHIELD=arceli_eth_w5500`` when you invoke ``west build``. For example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/dhcp_client
+   :board: nrf52840dk_nrf52840
+   :shield: arceli_eth_w5500
+   :goals: build
```
Comment (mbolivar-nordic) [inline boards/shields/arceli_eth_w5500/doc/index.rst:48]: The example does not match the description. The SHIELD value is not set. What am I missing?

**Review by mbolivar-nordic — CHANGES_REQUESTED** [review]

File: boards/shields/arceli_eth_w5500/doc/index.rst (hunk 1 above)
Comment (parthitce) [reply boards/shields/arceli_eth_w5500/doc/index.rst:48]: @mbolivar-nordic `SHIELD` value? 
```
:shield: arceli_eth_w5500
```
do you mean this part?

**Review by mbolivar-nordic — APPROVED** [review]
Thanks, sorry

**Pull request merged**